.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
```

Comment:
Check out these .gitignores. You may want to combine them both

https://github.com/github/gitignore/blob/main/Python.gitignore
https://github.com/github/gitignore/blob/main/Terraform.gitignore